package.json

```diff
@@ -36,39 +36,37 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
-    "debug": "^4.1.1",
-    "ioredis": "^4.17.3",
-    "lodash": "^4.17.15",
-    "moment": "^2.26.0",
-    "p-queue": "^6.4.0",
-    "p-timeout": "^3.2.0",
-    "rate-limiter-flexible": "^2.1.7",
-    "set-interval-async": "^1.0.32",
-    "uuid": "^8.1.0"
+    "dayjs": "1.11.5",
+    "debug": "4.3.4",
+    "ioredis": "5.2.3",
+    "p-queue": "6.6.2",
+    "p-timeout": "4.1.0",
+    "rate-limiter-flexible": "2.3.12",
+    "set-interval-async": "3.0.2",
+    "uuid": "9.0.0"
```

Comment:
I don't think `dependencies` in "library"-like packages should be fixed to exact versions. Could we still have carets here, please?

Reply:
Isn't that the whole point of have proper versions and to keep compatibility? Caret may install a different version than what you see.

Reply:
caret are minor + patch. Maybe no breaking changes if every lib are following the semantics.